http/sys_in_flight_requests.go

```diff
@@ -0,0 +1,45 @@
+package http
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/hashicorp/vault/vault"
+)
+
+func handleUnAuthenticatedInFlightRequest(core *vault.Core) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		switch r.Method {
+		case "GET":
+		default:
+			respondError(w, http.StatusMethodNotAllowed, nil)
+			return
+		}
+		now := time.Now()
+		currentInFlightReqMap := make(map[string]interface{})
+		syncMapRangeResult := true
+		core.RangeInFlightReqData(func(key, value interface{}) bool {
+			v, ok := value.(*vault.InFlightReqData)
+			if !ok {
+				syncMapRangeResult = false
+				return false
+			}
+			// don't report the request to the in-flight-req path
+			if v.ReqPath != "/v1/sys/in-flight-req" {
```

Comment:
Why should we exclude these?

Reply:
I'd rather include everything.  It would be ironic if a problem caused by this endpoint couldn't be debugged because we filtered out the relevant requests.

Reply:
Done